### DISCUSSION TITLE
Ignoring folders

### DISCUSSION THREAD

**Discussion by MyWebmasterD** [photoprism/photoprism#4536] [Q&A] [answered]
Hi guys,

do you understand how to ignore folders? Instructions are not so clear..

I want to ignore all the folders named Trash, independently from the folder structure.

Should I use [Trash] or something like *[Trash]* in the ignore file?

**Comment by DJKeterson** [comment]
How about you simply remove those folders from your originals or import folder? What are you trying to achieve?

**Reply by MyWebmasterD** [reply]
I can't remove my capture one original folders, I'm a professional photographer and in my media folder I store all the projects of my sessions.
Trash was only an example, I also don't want to index other folders

**Reply by MyWebmasterD** [reply]
seems that using Trash without [ ] worked well!

**Comment by graciousgrey** [comment]
> I want to ignore all the folders named Trash, independently from the folder structure.

`*` is used as wildcard character in .ppignore files. So adding `Trash` and `*Trash*` should work.

**Reply by MyWebmasterD** [reply]
sorry github removed my ** , I indeed used * [ Trash ] * (without any spaces) but the folders were indexed anyway

**Reply by MyWebmasterD** [reply, marked as answer]
ok seems that using Trash without [ ] worked well!
